pandas/core/config.py

```diff
@@ -196,7 +196,10 @@ def __getattr__(self, key):
         if prefix:
             prefix += "."
         prefix += key
-        v = object.__getattribute__(self, "d")[key]
+        d = object.__getattribute__(self, "d")
+        if key not in d:
```

Comment:
slightly more idiomatic to use a try/except here (to catch the KeyError) and raise the OptionError

Reply:
Done